roles/openshift_node_group/tasks/create_config.yml

```diff
@@ -13,7 +13,7 @@
   run_once: true
 
 - when:
-  - configout.results.results.0 == {} or (configout.results.results.0 != {} and openshift_node_group_edits|length > 0)
+  - configout.results.results.0 == {} or (configout.results.results.0 != {} and (openshift_node_group_edits|length > 0 or openshift_node_group_labels|length > 0))
```

Comment:
This statement will never be true.

Reply:
These get set by main.yml when iterating over openshift_node_groups.